### PR TITLE
add FROST DAG; revamp index

### PR DESCRIPTION
**Pull request by conradoplg**
- add a DAG for the FROST project
- add a stylesheet to index to make it a bit prettier, and reword it to focus on ZF DAGs

### CODE REVIEW

**Comment by conradoplg** [comment]
Wrong repo, sorry :upside_down_face: (we're now using our fork)

**Comment by str4d** [comment]
Just to be clear, this is not intended to be an ECC-specific repository, so if there are general changes you want to upstream, I'm happy to have them.

**Comment by conradoplg** [comment]
> Just to be clear, this is not intended to be an ECC-specific repository, so if there are general changes you want to upstream, I'm happy to have them.

@str4d Thanks, we appreciate that! We just wanted to get some new diagrams and change CI (to generate it hourly), and I did that in a fork to reduce friction (I didn't want to get you distracted with our stuff, and we wanted to get those in soon). I think it may make sense to keep them separated from now on, but if you're interested I can definitely create a PR upstream.
